_includes/known-issues-alert.html

```diff
@@ -1,4 +1,4 @@
-<div class="usa-alert margin-bottom-3 usa-alert--{{ known_issues.alert_type }}">
+<div class="usa-alert site-alert margin-bottom-3 usa-alert--{{ known_issues.alert_type }}">
```

Comment:
Use the same pattern we've used before for styling components we need to customize on our site